packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -1001,38 +1018,38 @@ export const mixinRequestHandler = (
  * @param Base - base class, inherits from FluidDataStoreRuntime
  */
 export const mixinSummaryHandler = (
-    handler: (runtime: FluidDataStoreRuntime) => Promise<{ path: string[]; content: string; } | undefined >,
+    handler: (runtime: FluidDataStoreRuntime) => Promise<{ path: string[]; content: string; } | undefined>,
     Base: typeof FluidDataStoreRuntime = FluidDataStoreRuntime,
 ) => class RuntimeWithSummarizerHandler extends Base {
-        private addBlob(summary: ISummaryTreeWithStats, path: string[], content: string) {
-            const firstName = path.shift();
-            if (firstName === undefined) {
-                throw new LoggingError("Path can't be empty");
-            }
+    private addBlob(summary: ISummaryTreeWithStats, path: string[], content: string) {
```

Comment:
I'd suggest to submit formatting changes in separate PRs. It's makes tracking history harder

Reply:
I agree with you, but it gets annoying to enable/disable code formatting in vscode for files which are out of sync. Perhaps we should format everything and make this a linter error.